File: js/day-1.js
const captcha = (input) => {
    const inputArray = [...input, input[0]];
    return inputArray.reduce((sum, val, i) => {
        if (parseInt(inputArray[i]) === parseInt(inputArray[i+1])) {
            return parseInt(val) + sum;
            }
        return sum;
    }, 0);
};

File: python/day-1.py
inputFile = __file__.replace('py', 'txt')

def getInput(fileName):
    with open(fileName) as f:
        return f.readline()

def captchaCalc():
    sum = 0
    input = getInput(inputFile)
    input += input[0]
    for i in range(len(input) - 1):
        if input[i] == input[i+1]:
            sum += int(input[i])
    return sum



print(captchaCalc())